engineering/workflow.md

```diff
@@ -54,31 +55,153 @@ The team conducts a Sprint Planning meeting for 60 minutes at the beginning of e
 The goal of this meeting is to review our major work items, synchronize with one another, and prioritize work across team members.
 
 See [the Sprint Planning issue template](https://github.com/2i2c-org/team-compass/blob/main/.github/ISSUE_TEMPLATE/meeting-sprint-planning.md) for the agenda / structure of these meetings.
- 
+
 (coordination:sprint-board)=
 ### The Sprint Board
 
-The Sprint Board is a place to keep track of the [Deliverables and tasks](coordination:deliverables) our team intends to work on for the week.
-It is a GitHub Projects board that is created for each week, and closed at the end of each week.
+The [Sprint Board](https://github.com/orgs/2i2c-org/projects/49/views/1) is a place to keep track of the [Deliverables and tasks](coordination:deliverables) our team intends to work on for a two week iteration.
+
+The board is a GitHub Projects board that is populated with with work items during the teams Itereation Planning activity.
+
 The team's goal is to complete all items on the Sprint Board by the end of the Sprint.
 This is a team commitment - while one person may be assigned to a deliverable, we all commit to working together to get the work done.
 
 :::{admonition} The Sprint Board should...
 :class: tip
-- Have enough items to keep the team occupied for the week
+- Be driven by 2i2c's product roadmap
+- With the exception of support tickets, work items should be related back to Initiatives.
```

Comment:
Changing the sentence order to follow the imperative "The Spring Board should ..."
```suggestion
- Contain work items that relate back to Initiatives (with the exception of support tickets).
```

Reply:
```suggestion
- Contain work items that relate back to Initiatives (with the exception of support tickets and new cluster/hub requests).
```

I think also new cluster/hub requests are an exception right?

Reply:
I'm unsure. There may be instances where a request is tied to an initiative and there are times it may not.

Reply:
I think it's safe to include the exception and remove it later if this proves to be false.